lib/src/main/java/com/basistheory/android/view/CardExpirationDateElement.kt

```diff
@@ -0,0 +1,66 @@
+package com.basistheory.android.view
+
+import android.content.Context
+import android.text.Editable
+import android.util.AttributeSet
+import com.basistheory.android.model.ElementValueReference
+import com.basistheory.android.model.KeyboardType
+import com.basistheory.android.view.validation.futureDateValidator
+
+class CardExpirationDateElement : TextElement {
+
+    constructor(context: Context) : super(context) {
+        init()
+    }
+
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        init()
+    }
+
+    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
+        context,
+        attrs,
+        defStyleAttr
+    ) {
+      init()
+    }
+
+    fun month(): ElementValueReference = ElementValueReference(::getMonthValue)
+
+    fun year(): ElementValueReference = ElementValueReference(::getYearValue)
+
+    private fun init() {
+        super.keyboardType = KeyboardType.NUMBER
+        super.mask = defaultMask
+        super.validate = { futureDateValidator(it) }
+    }
+
+    private fun getMonthValue(): String? =
+        getText()
+            ?.split("/")
+            ?.elementAtOrNull(0)
+
+    private fun getYearValue(): String? =
+        getText()
+            ?.split("/")
+            ?.elementAtOrNull(1)
```

Comment:
Incorrect! I need to prepend "20" to the year or else this won't be usable to create a card token